core/jazz_ui/src/app/core/services/request.service.ts

```diff
@@ -164,6 +164,48 @@ export class RequestService {
             })
 
     }
+    delete(url: string, body: any, serviceId?): Observable<any> {
+        // Make a PUT request to url
+
+        // Construct url
+        url = this.constructUrl(url);
+
+        // Get Authentication token
+        this.token = this.authenticationService.getToken();
+
+        // Add Authentication token to headers
+        let headerObj = {
+            'Authorization': this.token,
+            'Content-Type': 'application/json',
+            'accept': 'application/json'
+        };
+
+        if (serviceId) {
+            headerObj['Jazz-Service-ID'] = serviceId
+        }
+
+        let headers = new Headers(headerObj);
+        let options = new RequestOptions({ headers: headers,body:JSON.stringify(body) });
+        let router = this.router;
+
+        return this.http.delete(url, options)
+            .map((response: Response) => {
+                let responseBody;
+                responseBody = response.json();
+                if (responseBody) {
+
```

Comment:
Nit: please maintain formatting standards in lieu of lint.

Reply:
done

Reply:
done